tests/console/systemd_testsuite.pm

```diff
@@ -28,6 +29,12 @@ sub run {
     my $package_name = 'systemd-v$systemd_version-testsuite';
     assert_script_run 'zypper -n se -sx ' . $package_name;
     zypper_call 'in ' . $package_name;
+    if (is_leap('16.0+') || is_tumbleweed || is_sle && sle_version_at_least('16')) {
+        # Systemd has split its nspawn feature to a systemd-container package.
+        #   - This change is not included in SLE15 nor Leap15.
+        record_soft_failure('poo#32614 - Missing systemd-container');
```

Comment:
basically you unconditionally record softfailure based on detected version. So when/if  systemd-container will be added to target versions you will keep failing , please check that it's not installed before record softfailure

Reply:
There is no systemd-container on lower versions. So the package is always uninstalled. Checking if it is not installed would make no sense. And this is a workaround until tblume fixes the testsuite so it install by itself.